src/calculateRentalCost.js

```diff
@@ -28,7 +28,14 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = 40 * days;
```

Comment:
Consider using a constant variable for the daily rental cost (40) instead of a magic number.